cmd/iam.go

```diff
@@ -1114,7 +1118,24 @@ func (sys *IAMSys) NewServiceAccount(ctx context.Context, parentUser string, ses
 
 	cr, ok := sys.iamUsersMap[parentUser]
 	if !ok {
-		return auth.Credentials{}, errNoSuchUser
+		// For LDAP users we would need this fallback
```

Comment:
So if I understood correctly, parentUser in LDAP doesn't need to exist in MinIO backend but always present as a claim in requests ?

Reply:
correct @vadmeste parentUser never exists in `UsersMap` for LDAP